src/dogstatsd_worker.erl

```diff
@@ -33,9 +33,10 @@ init([]) ->
     State = case stillir:get_config(dogstatsd, send_metrics) of
                 true ->
                     {ok, Socket} = gen_udp:open(0),
+                    Ip = inet:getaddr(stillir:get_config(dogstatsd, agent_port), inet),
```

Comment:
think you meant (agent_port -> agent_address)
`Ip = inet:getaddr(stillir:get_config(dogstatsd, agent_address), inet),`